src/table/__integ__/resizable-columns.test.ts

```diff
@@ -334,6 +334,11 @@ test(
 test(
   'should recover column widths when the inner state is reset',
   setupTest(async page => {
+    // TODO: fix test for React 18
+    if ((await page.getReactVersion()) === '18') {
```

Comment:
Is there any reason why you skip in runtime instead of doing something like `REACT_VERSION === '18' ? test.skip : test`?